src/Outlines.js

```diff
@@ -426,8 +427,48 @@ export default class Outlines {
 			return open + info.label + " " + info.qualifiedNumber + groups.close;
 		});
 
-		// TODO replace links to other files in the outline with local links
-		// (whether they are empty or not)
+		if (urls) {
+			// The outline spans multiple URLs
+			// Replace links to other files in the outline with local links
+			// (whether they are empty or not)
+			content = content.replaceAll(linkRegex, (match, ...args) => {
+				let groups = processGroups(args.at(-1));
+				let attrs = groups.attrs;
+				attrs = html.parseAttributes(attrs);
+
+				let href = attrs.href;
+				let classes = attrs.class ?? "";
+
+				let ignore = href.startsWith("#") || classes.includes("outline-ignore");
```

Comment:
I wonder if it makes sense to do something like this instead? 🤔

```js
let classes = (attrs.class ?? "").split(/[\s\r?\n?]+/).filter(Boolean);
classes = new Set(classes);

let ignore = href.startsWith("#") || classes.has("outline-ignore");
```

Reply:
or just `/(?:^|\s)outline-ignore(?:$|\s)/.test(classes)`

Reply:
Love it! Will do it this way. Thanks!

Reply:
Done!